portfolio/src/main/webapp/tasksApi.js

```diff
@@ -102,6 +54,11 @@ function importTasklist(tasklistId) {
  * logged in.
  */
 function drawImportMenu() {
+  importMenuVisible = true;
+
+  const $toggleTasksButton = $('#connect-tasks-btn')[0];
```

Comment:
Is this a jquery variable? If not, I think it shouldn't start with $

Reply:
Done.